### PR TITLE
Add pop-count exercise

### PR DESCRIPTION
**Pull request by pfertyk**

### CODE REVIEW

**Comment by pfertyk** [comment]
@BNAndras the example solution uses a built-in method to convert a number to binary. Please let me know if I should instead implement the conversion step-by-step.

**Comment by BNAndras** [comment]
Should be fine. The example just needs to prove the exercise can be solved.

File: exercises/practice/pop-count/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "authors": [],
```
Comment (BNAndras) [inline exercises/practice/pop-count/.meta/config.json:2]: Add GitHub username

Comment (pfertyk) [reply exercises/practice/pop-count/.meta/config.json]: I keep forgetting about this, thanks :D

**Comment by BNAndras** [comment]
Looks good. I'd rebase the PR to resolve the merge conflict

**Pull request merged**